### PR TITLE
wcms-1025 - improve head section implementation

### PR DESCRIPTION
**Pull request by DaRaFF**
after a first hack in https://github.com/nzzdev/livingdocs-render-decorator/pull/22 I decided to make the fields enriched hidden in the render decorator more visible in the editor (after feedback from @sshobotov ).

The editor can now see, where specific fields are placed to (flag, publicationdate, comments). Its just a placeholder, which will be enriched during publishing with the render decorator. It's not perfect, but better than seeing nothing.

empty-new-article

### CODE REVIEW

**Comment by DaRaFF** [comment]
@sshobotov pls do a review, thx!

File: src/middleware/post-publicationdate-adder.coffee [hunk 1]
```diff
@@ -5,21 +5,28 @@ momentTimezone = require('moment-timezone')
...
-    unformattedPublicationDate = momentTimezone(rawPublicationDate).tz('Europe/Zurich')
-    formattedPublicationDate = unformattedPublicationDate.format('D.M.YYYY, HH:mm')
-
     $ = cheerio.load(html)
+    rawPublicationDate = params?.metadata?.publicationDate || ""
 
-    $('.metainfo__item-author').each (index, element) ->
-      $parent = $(element).parent()
-      $parent.after("<li class=\"metainfo__item\"><time class=\"metainfo__item-date\" datetime=\"#{rawPublicationDate}\">#{formattedPublicationDate} Uhr</time></li>");
+    if momentTimezone(rawPublicationDate).isValid() is false
```
Comment (sshobotov) [inline src/middleware/post-publicationdate-adder.coffee]: In this case I think old plain "if else" will be more readable

**Comment by sshobotov** [comment]
@DaRaFF Review done, nice code

File: src/middleware/post-publicationdate-adder.coffee (hunk 1 above)
Comment (DaRaFF) [inline src/middleware/post-publicationdate-adder.coffee]: right, fixed

**Pull request merged**